linux/src/threadpool_linux.c

```diff
@@ -8,6 +8,14 @@
 #include <semaphore.h>
 #include <time.h>
 #include <string.h>
+#include <signal.h>
+#include <errno.h>
+
+#include <bits/types/__sigval_t.h>         // for __sigval_t
```

Comment:
>__sigval_t [](http://example.com/codeflow?start=50&length=10)

404 not found.

Reply:
iwyu demands all the bits/types includes.  I'm not sure if this is correct or not.